charts/rhtap-tpa/templates/realm.yaml

```diff
@@ -53,6 +63,47 @@ data:
     {{- $oidcCredentials := mergeOverwrite $oidcCredentials (dict $k $s) }}
   {{ $k }}: {{ $s }}
   {{- end }}
+#
+# Integration Secret
+#
+#   Creates a secret to store integration details.
+#
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: keycloak
```

Comment:
This value doesn't seem relevant. Should it go away or be replaced with `trustification`?